tests/fixtures/assets/app.scss

```diff
@@ -0,0 +1,3 @@
+p {
+  color: red;
+}
```

Comment:
Is this meant to be here? I'm asking because `FunctionalTest` seems to create this in its `setUp()`.

Also, if we set a variable then use it, the assertion in the test will be a bit more meaningful, as we'll see Sass doing its job.

Reply:
Yes this is for the SassBuilderTest